### PR TITLE
@users/gage/feedmewords mobile

### PR DESCRIPTION
**Pull request by inquisitorPinkie**

### CODE REVIEW

File: src/services/app/data_service.ts [hunk 1]
```diff
@@ -8,23 +8,28 @@ type Firestore = firebase.firestore.Firestore;
 export class DataService {
   private _db?: Firestore;
 
-  get db(): Firestore {
-    return this._db || this.init();
+  get db(): Firestore | null {
+    return this._db ?? this.init() ?? null;
```
Comment (prmichaelsen) [inline src/services/app/data_service.ts:12]: Why was this change needed?

**Comment by inquisitorPinkie** [comment]
It probably isn't needed. It was one of the first things I changed and I
forgot to undo it when I made the other changes too.
